src/drivers/uavcan/sensors/battery.cpp

```diff
@@ -191,7 +191,10 @@ UavcanBatteryBridge::battery_aux_sub_cb(const uavcan::ReceivedDataStructure<ardu
 		_battery_status[instance].voltage_cell_v[i] = msg.voltage_cell[i];
```

Comment:
Note to myself:
`_battery_status[instance].time_remaining_s = math::isZero(_battery_status[instance].current_a) ? NAN :`

Reply:
Done in place now with 8df6d3cbb330995516c862c6c6fcae9f95306314